test/pageobjects/BookStore.js

```diff
@@ -0,0 +1,42 @@
+class BookStore {
+  get bookStoreBtn() {
+    return $("#item-2*=Book Store");
+  }
+  get profileBtn() {
+    return $("#item-3*=Profile");
+  }
+  get bookGitGuide() {
+    return $("=Git Pocket Guide");
+  }
+  get addToCollectionBtn() {
+    return $$("#addNewRecordButton")[1];
```

Comment:
Is this really working in terms of using async method?

Reply:
What do you mean? Can you be more specific please?